doc/Learn/Tutorials/DependencyInjection/HowTo-CommunityToolkit.md

```diff
@@ -3,9 +3,6 @@ uid: Learn.Tutorials.DependencyInjection.HowToCommunityToolkit
 ---
 ## CommunityToolkit.Mvvm
 
-> [!TIP]
```

Comment:
Do we not need this tip any more? If not, how do users get started with adding appBuilder?